backend/src/config/database.ts

```diff
@@ -0,0 +1,25 @@
+import { Sequelize } from "sequelize";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+export const sequelize = new Sequelize(process.env.DATABASE_URL as string,
+    {
+        dialect: "postgres",
```

Comment:
Take dialect also from env. Since Sequelize can be used with any dialect. So, we should make the code generic to run on any database.